client/packages/common/src/ui/forms/JsonForms/useJsonForms.tsx

```diff
@@ -80,9 +80,16 @@ const ScrollFix = () => {
   return null;
 };
 
+export type SaveJob = (
```

Comment:
Nit, but not a huge fan of the name `SaveJob`. 

I think it could just be "saveData", and then you can rename the existing internal "saveData" method to "handleSave", which follows a fairly standard convention.

Reply:
thanks, didn't like the name neither. I renamed SaveJob -> SaveDocumentMutation and saveData -> handleSave